### PR TITLE
Moving bower dependencies to npm.

### PR DESCRIPTION
**Pull request by gmurphey**
I wasn't able to move `font-awesome` to npm, as that package isn't valid for `require.resolve`.

### CODE REVIEW

**Comment by jasonmit** [comment]
@gmurphey looks good just needs a rebase and sorry for not merging this ahead of the other

File: blueprints/ember-bootstrap-datetimepicker/index.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-module.exports = {
-  normalizeEntityName: function() {},
-
-  afterInstall: function() {
-    return this.addAddonToProject('ember-cli-moment-shim', '~0.6.1').then(function() {
```
Comment (jasonmit) [inline blueprints/ember-bootstrap-datetimepicker/index.js]: I think we want to keep this

**Comment by gmurphey** [comment]
@jasonmit no problem, gave me a chance to improve upon it from some of your suggestions in https://github.com/jasonmit/ember-cli-moment-shim/pull/22. :smile:

**Comment by jasonmit** [comment]
Looks good, thanks :)

**Comment by jasonmit** [comment]
I'll let @claudiocro version this

**Pull request merged**